readme.md

```diff
@@ -38,74 +30,32 @@ gulp.task('default', () => {
 
 Comes bundled with the following **lossless** optimizers:
 
-- [gifsicle](https://github.com/kevva/imagemin-gifsicle) — *Compress GIF images*
-- [jpegtran](https://github.com/kevva/imagemin-jpegtran) — *Compress JPEG images*
-- [optipng](https://github.com/kevva/imagemin-optipng) — *Compress PNG images*
-- [svgo](https://github.com/kevva/imagemin-svgo) — *Compress SVG images*
+- [gifsicle](https://github.com/imagemin/imagemin-gifsicle) — *Compress GIF images*
+- [mozjpeg](https://github.com/imagemin/imagemin-mozjpeg) — *Compress JPEG images*
+- [optipng](https://github.com/imagemin/imagemin-optipng) — *Compress PNG images*
+- [svgo](https://github.com/imagemin/imagemin-svgo) — *Compress SVG images*
 
-### imagemin([options])
+### imagemin([plugins], [options])
 
 Unsupported files are ignored.
 
-#### options
-
-Options are applied to the correct files.
-
-##### optimizationLevel *(png)*
-
-Type: `number`  
-Default: `3`
-
-Select an optimization level between `0` and `7`.
-
-> The optimization level 0 enables a set of optimization operations that require minimal effort. There will be no changes to image attributes like bit depth or color type, and no recompression of existing IDAT datastreams. The optimization level 1 enables a single IDAT compression trial. The trial chosen is what. OptiPNG thinks it’s probably the most effective. The optimization levels 2 and higher enable multiple IDAT compression trials; the higher the level, the more trials.
-
-Level and trials:
-
-1. 1 trial
-2. 8 trials
-3. 16 trials
-4. 24 trials
-5. 48 trials
-6. 120 trials
-7. 240 trials
-
-##### progressive *(jpg)*
-
-Type: `boolean`  
-Default: `false`
-
-Lossless conversion to progressive.
+#### plugins
 
-##### interlaced *(gif)*
+Type: `array`<br>
+Default: `[imageminGifsicle(), imageminMozjpeg(), imageminOptipng({optimizationLevel: 3}), imageminSvgo({multipass: true})]`
```

Comment:
Prefix all the methods with `imagemin.`, so it will work in case the user chooses to copy-paste.